build.gradle

```
@@ -22,7 +22,7 @@ plugins {
 }
 
 javafx {
-	version = '17'
+	version = '18'
```

We officially only support Java 11 and 17
```suggestion
	version = '17'
```